conf/config.yaml

```diff
@@ -30,6 +30,16 @@
 # And then run `export ETCD_HOST=$your_host` before `make init`.
 #
 # If the configured environment variable can't be found, an error will be thrown.
+#
+# Also, If you want to use default value when the environment variable not set,
+# Use `${{VAR::default_value}}` instead. For instance:
```

Comment:
I am curious why `::` is chosen as the separator. It is different from the common way. Is there a special reason behind it?

Reply:
I want to use `:` in first time, But sometimes default value may contains it, like etcd.host=${{ETCD_HOST::demo.domain.com:1234}}

Reply:
What about using `:=`? In bash, it means "Assign  Default Values".

Reply:
Done